k2/python/csrc/version.cu

```diff
@@ -30,4 +31,5 @@ void PybindVersion(py::module &m) {
   version.attr("torch_version") = k2::kTorchVersion;
   version.attr("torch_cuda_version") = k2::kTorchCudaVersion;
   version.attr("enable_nvtx") = k2::kEnableNvtx;
+  version.attr("disable_debug") = k2::internal::kDisableDebug;
```

Comment:
Does EnableCudaDeviceSync() or K2_SYNC_KERNELS get printed out in the benchmarks?

Reply:
NO, `version.cu` is for collecting environment information in which k2 was built. This is helpful
when reporting issues.

Users can run
```
python3 -m k2.version
```
and it will print something like the following:
(copied from https://github.com/k2-fsa/k2/runs/1567209962?check_suite_focus=true#step:13:12)

```
Collecting environment information...

k2 version: 0.1.1
Build type: Release
Git SHA1: cfe8382205e27ccbd2347c0271848463298e1807
Git date: Wed Dec 16 23:43:25 2020
Cuda used to build k2: 10.1
cuDNN used to build k2: 8.0.2
Python version used to build k2: 3.6
OS used to build k2: Ubuntu 18.04.5 LTS
CMake version: 3.19.1
GCC version: 5.5.0
CMAKE_CUDA_FLAGS:  -D_GLIBCXX_USE_CXX11_ABI=0  --expt-extended-lambda -gencode arch=compute_35,code=sm_35 --expt-extended-lambda -gencode arch=compute_50,code=sm_50 --expt-extended-lambda -gencode arch=compute_60,code=sm_60 --expt-extended-lambda -gencode arch=compute_61,code=sm_61 --expt-extended-lambda -gencode arch=compute_70,code=sm_70 --expt-extended-lambda -gencode arch=compute_75,code=sm_75
CMAKE_CXX_FLAGS:  -D_GLIBCXX_USE_CXX11_ABI=0
PyTorch version used to build k2: 1.6.0+cu101
PyTorch is using Cuda: 10.1
NVTX enabled: True
```

---

Will append a commit to this pullrequest to print it for benchmarks.

Reply:
Fixed